openslides_backend/action/actions/option/update.py

```diff
@@ -0,0 +1,173 @@
+from typing import Any, Dict, List, Optional, Tuple
+
+from ....models.models import Option
+from ....services.datastore.commands import GetManyRequest
+from ....shared.exceptions import ActionException
+from ....shared.patterns import Collection, FullQualifiedId
+from ...generics.update import UpdateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ..vote.create import VoteCreate
+from ..vote.update import VoteUpdate
+
+
+@register_action("option.update")
+class OptionUpdateAction(UpdateAction):
+    """
+    Action to update a user.
+    """
+
+    model = Option()
+    schema = DefaultSchema(Option()).get_update_schema(
+        additional_optional_fields={
+            "Y": {"type": "string", "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$"},
+            "N": {"type": "string", "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$"},
+            "A": {"type": "string", "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$"},
```

Comment:
Please use the decimal schema here that you created in another PR.

Reply:
Include the decimal schema in schema and use it here.